src/components/Timeline.vue

```diff
@@ -240,7 +240,7 @@ export default class Timeline extends Mixins(GlobalMixin, UserConfig) {
 
   @Watch("$route")
   async routeChange(to: any, from?: any) {
-    if (from?.path !== to.path) {
+    if (from?.path !== to.path || from.query.recursive !== to.query.recursive) {
```

Comment:
Maybe for the query in general rather than recursive specifically?

Reply:
Sure, I can change it to `from.query !== to.query` which also works, or do you have a suggestion for another way?

Reply:
That sounds good to me, so long as it works.